### PR TITLE
USHIFT-459: Add hostport to ingress

### PR DESCRIPTION
**Pull request by fzdarsky**
Signed-off-by: Frank A. Zdarsky <fzdarsky@redhat.com>

Closes [USHIFT-459](https://issues.redhat.com//browse/USHIFT-459)

### CODE REVIEW

File: assets/components/openshift-router/deployment.yaml [hunk 1]
```diff
@@ -87,12 +87,15 @@ spec:
...
               protocol: TCP
+              hostPort: 80
             - name: https
               containerPort: 443
               protocol: TCP
+              hostPort: 443
             - name: metrics
               containerPort: 1936
               protocol: TCP
+              hostPort: 1936
```
Comment (mangelajo) [inline assets/components/openshift-router/deployment.yaml:98]: We should probably not export the metrics on the host.

File: scripts/auto-rebase/rebase.sh [hunk 2]
```diff
@@ -557,10 +557,10 @@ update_manifests() {
...
     # yq -i '.spec.template.spec.hostNetwork = "true"' "${REPOROOT}"/assets/components/openshift-router/deployment.yaml
-    #    Add hostPorts for routes and metrics
-    # yq -i '.spec.template.spec.containers[0].ports[0].hostPort = 80' "${REPOROOT}"/assets/components/openshift-router/deployment.yaml
-    # yq -i '.spec.template.spec.containers[0].ports[1].hostPort = 443' "${REPOROOT}"/assets/components/openshift-router/deployment.yaml
-    # yq -i '.spec.template.spec.containers[0].ports[2].hostPort = 1936' "${REPOROOT}"/assets/components/openshift-router/deployment.yaml
+    #    Add hostPorts for routes and metrics (needed as long as there is no load balancer)
+    yq -i '.spec.template.spec.containers[0].ports[0].hostPort = 80' "${REPOROOT}"/assets/components/openshift-router/deployment.yaml
+    yq -i '.spec.template.spec.containers[0].ports[1].hostPort = 443' "${REPOROOT}"/assets/components/openshift-router/deployment.yaml
+    yq -i '.spec.template.spec.containers[0].ports[2].hostPort = 1936' "${REPOROOT}"/assets/components/openshift-router/deployment.yaml
```
Comment (mangelajo) [inline scripts/auto-rebase/rebase.sh:563]:
```suggestion
```

File: assets/components/openshift-router/deployment.yaml (hunk 1 above)
Comment (mangelajo) [inline assets/components/openshift-router/deployment.yaml:98]:
```suggestion
```

**Review by mangelajo — CHANGES_REQUESTED** [review]
check the comments

File: assets/components/openshift-router/deployment.yaml (hunk 1 above)
Comment (fzdarsky) [reply assets/components/openshift-router/deployment.yaml]: Ack, removed.

File: scripts/auto-rebase/rebase.sh (hunk 2 above)
Comment (fzdarsky) [reply scripts/auto-rebase/rebase.sh]: Removed.

File: assets/components/openshift-router/deployment.yaml (hunk 1 above)
Comment (fzdarsky) [reply assets/components/openshift-router/deployment.yaml]: Removed.

**Review by mangelajo — COMMENTED** [review]
/lgtm

**Pull request merged**